src/components/title/show/title-show.js

```diff
@@ -295,20 +295,23 @@ class TitleShow extends Component {
               onSubmit={handleSubmit}
               wrappingElement="form"
               footer={(
-                <ModalFooter
-                  primaryButton={{
-                    'label': request.isPending ? (modalMessage.saving) : (modalMessage.submit),
-                    'type': 'submit',
-                    'disabled': request.isPending,
-                    'data-test-eholdings-custom-package-modal-submit': true
-                  }}
-                  secondaryButton={{
-                    'label': <FormattedMessage id="ui-eholdings.cancel" />,
-                    'onClick': this.toggleCustomPackageModal,
-                    'disabled': request.isPending,
-                    'data-test-eholdings-custom-package-modal-cancel': true
-                  }}
-                />
+                <ModalFooter>
+                  <Button
+                    data-test-eholdings-custom-package-modal-submit
+                    buttonStyle="primary"
+                    disabled={request.isPending}
+                    type="submit"
+                  >
+                    {request.isPending ? (modalMessage.saving) : (modalMessage.submit)}
```

Comment:
Parenthesis make no sense here. Please, delete them.

Reply:
Commit added with that change.